package.json

```diff
@@ -22,8 +22,8 @@
     "knip-ci": "knip --no-exit-code --reporter json",
     "prepare": "husky install",
     "build:types": "tsc --emitDeclarationOnly --declaration --outDir dist && rm -rf dist/src",
-    "build:tests": "tsx build/esbuild-build-tests.ts && tsc --emitDeclarationOnly --declaration --project tsconfig.tests.json",
-    "build": "rm -rf dist && tsx build/esbuild-build.ts",
+    "build:tests": "tsx build/types.ts && tsx build/esbuild-build-tests.ts && tsc --emitDeclarationOnly --declaration --project tsconfig.tests.json",
+    "build": "rm -rf dist && tsx build/types.ts && tsx build/esbuild-build.ts ",
     "postbuild": "yarn build:types",
```

Comment:
You're building types twice?

Also consider using the run-s syntax because we include that tool in our template. 

For example

"run-s clean:dist build:types build"

And each of those steps should be defined in other package.json scripts 

You may be able to review our other repos for examples of using run-s from the npm-run-all dependency.

Reply:
I will improve the naming convention or combine the two. One is building the dynamic types and const data used throughout which `postbuild` would then bundle into the final build types.

I've had issues with run-s in other repos before which is why it's not my go-to but I will update it as it is a cleaner approach